deploy/helm/kubecf/values.yaml

```diff
@@ -511,7 +507,7 @@ bits:
   kube:
     external_ips: []
   services:
-    loadbalanced: true
+    nodePort: 31666
```

Comment:
Should we not have a doc comment to mark this hidden?

Reply:
I was unsure how the tooling handles the sub-chart configuration.
Will fix.

Reply:
Done in commit 56f87f06fdedf93929d43f196af36084bc48be5f